### PR TITLE
making Makefile code a bit more DRY

### PR DESCRIPTION
**Pull request by elreydetoda**
related to #396

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,9 +1,10 @@
-dev:
+dev-build:
 	docker build --rm -f Dockerfile.dev -t jb_hugo:latest .
```
Comment (elreydetoda) [inline Makefile:2]: pulling out the build definition from each of the subsequent dev definitions allows you to re-use this step in other places.

File: Makefile [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+
+dev: dev-build
```
Comment (elreydetoda) [inline Makefile:4]: this tells make, if you're going to do this step then `dev-build` is a dependency of this step.

File: Makefile [hunk 3]
```diff
@@ -1,9 +1,10 @@
...
 	docker run --rm -it -v "$${PWD}":/app -w /app -p $${HOST_IP:-127.0.0.1}:1313:1313 jb_hugo:latest
 
-dev-build-metrics:
-	docker build --rm -f Dockerfile.dev -t jb_hugo:latest .
+dev-build-metrics: dev-build
```
Comment (elreydetoda) [inline Makefile:7]: (same thing as above) this tells make, if you're going to do this step then `dev-build` is a dependency of this step.

Comment (gerbrent) [reply Makefile:7]: nice!

**Review by gerbrent — APPROVED** [review]
tested and confirmed working as intended 👍🏻

**Pull request merged**